frontera/contrib/scrapy/schedulers/frontier.py

```diff
@@ -124,11 +134,13 @@ def process_exception(self, request, exception, spider):
     def open(self, spider):
         log.msg('Starting frontier', log.INFO)
         if not self.frontier.manager.auto_start:
-            self.frontier.start()
+            scrapy_kwargs = {'spider': spider}
+            self.frontier.start(**scrapy_kwargs)
```

Comment:
@kalessin Here is the idea how frontera options can be overwritten from command line.
``` scrapy crawl linkedin -s FRONTERA_HCF_CONSUMER_SLOT=linkedin1 ```
It sets Scrapy setting with prefix ``FRONTERA_`` which is detected by Frontera scheduler and set ``HCF_CONSUMER_SLOT`` setting in Frontera settings.
what do you think?